packages/protocol/test/governance/network/governance.ts

```diff
@@ -2568,6 +2593,51 @@ contract('Governance', (accounts: string[]) => {
     })
   })
 
+  describe('#dequeueProposalsIfReady()', () => {
+    it('should not update lastDequed when no queued proposal(s)', async () => {
+      const originalLastDequeue = await governance.lastDequeue()
+      await timeTravel(dequeueFrequency, web3)
+      await governance.dequeueProposalsIfReady()
+
+      assert.equal((await governance.getQueueLength()).toNumber(), 0)
+      assert.equal((await governance.lastDequeue()).toNumber(), originalLastDequeue.toNumber())
+    })
+
+    describe('when proposal exists', () => {
+      beforeEach(async () => {
+        await governance.propose(
+          [transactionSuccess1.value],
+          [transactionSuccess1.destination],
+          // @ts-ignore bytes type
+          transactionSuccess1.data,
+          [transactionSuccess1.data.length],
+          descriptionUrl,
+          { value: minDeposit }
+        )
+      })
+
+      it('should update lastDequed when proposal queued', async () => {
```

Comment:
Spelling. "when proposal queued" is not necessary since it's already mentioned in the `describe` context above.
```suggestion
      it('should update lastDequeue', async () => {
```